manifests/plugin_panels.pp

```diff
@@ -0,0 +1,22 @@
+# panels-specific functionality.
+# TODO: probably should move this file into its own puppet module
+# TODO: need to make config handling more generic, this should work with ANY plugin, and not be panels-specific
+
+class uber::plugin_panels (
+  $git_repo = "https://github.com/magfest/panels",
+  $git_branch = "master",
```

Comment:
How does this interact with the ``uber::plugins::extra_plugins`` setting in e.g. ``event-prime.yaml``?  Like, what would happen if we had two different values for the Git repos?

Since this just defines a puppet management class, I assume this is NOT required for all deploys which use the ``ubersystem-puppet`` repo.  Like, an event could install a set of plugins that doesn't include this one?  Or is this now required for every deploy using ``ubersystem-puppet``?  (I wouldn't have a real problem with that since ``ubersystem-puppet`` is pretty much just MAGFest at this point and Docker seems to be the way other events are deploying, but I figured it was worth asking.)